package.json

```diff
@@ -64,16 +68,24 @@
   },
   "devDependencies": {
     "@babel/core": "7.21.3",
+    "@react-native-community/datetimepicker": "7.6.1",
+    "@react-native-community/slider": "4.4.2",
```

Comment:
I don't understand why we have that ? (as devDependencies that make even less sense)

Reply:
thoses libraries are used only for the storybook thats why its devDependencies and i will got thoses errors if I remove them 

```js
Unable to resolve "@react-native-community/slider" from "node_modules/@storybook/addon-ondevice-controls/dist/types/Number.js"
```

```js
Android Bundling failed 6064ms (node_modules/expo-router/entry.js)
Unable to resolve "@react-native-community/datetimepicker" from "node_modules/react-native-modal-datetime-picker/src/DateTimePickerModal.android.js"
```